packages/react-next/RELEASE_NOTES.md

```diff
@@ -70,6 +70,11 @@ TODO: document any API or functionality changes
 
 - Updated enums to string union type: `PivotLinkFormat`, `PivotLinkSize`. (#13370)
 
+### SwatchColorPicker
+
+- Added an `onChange` prop and deprecated `onColorChanged`.
+- Deprecated `isControlled`.
```

Comment:
A couple extra notes. `id` was required before, so I'm not sure why indices were used to track selection, but it's not a bad idea to call out.

```suggestion
- Removed deprecated props `positionInSet` (use `ariaPosInSet`) and `setSize` (use `ariaSetSize`).
- Added an `onChange` prop and deprecated `onColorChanged`.
- Deprecated `isControlled`. Provide `selectedId` for controlled behavior and `defaultSelectedId` for uncontrolled behavior.
- Selection state is now tracked internally based on `IColorCellProps.id`, not item index. Ensure that all color cells have a unique `id` property.
```